src/translations/en.json

```diff
@@ -610,7 +610,11 @@
   "ProductOrderForm.selectDeliveryMethodOption": "Select delivery method",
   "ProductOrderForm.selectQuantityOption": "Select...",
   "ProductOrderForm.shippingOption": "Shipping",
+  "ProfilePage.customFieldsHeading": "User information",
```

Comment:
If heading is removed, remember to remove this too.

Reply:
Removed!